### DISCUSSION TITLE
Wich library GD or Imagemagik?

### DISCUSSION THREAD

**Discussion by yanaleksandrov** [spatie/image#124] [Q&A]
I liked the library. Tell me, I see that spatie uses GD or imagick for image management. What works by default?

**Comment by brandon-clair** [comment]
[v2](https://spatie.be/docs/image/v2/usage/basic-usage#content-selecting-a-driver) and [v3](https://spatie.be/docs/image/v3/usage/basic-usage#content-selecting-a-driver) use Imagick by default. The v1 documentation does not explicitly say, but I assume that examining the source would be enough to know.
